doc/conf.py

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) 2014 gocept gmbh & co. kg
 # See also LICENSE.txt
 
+import gocept.package.sphinxconf
+
+
 _year_started = 2014
 
-import gocept.package.sphinxconf
```

Comment:
Could it be that the `_year_started` is already needed by the import?
(I hope there was a reason why we did such a strange order.)

Reply:
Yes, we do ... We should actually switch away from `gocept.package` ...

Reply:
hm, I tried it actually and it worked. We just read in the `conf.py` when we call `set_defaults()`. So I will keep it like this.